scripts/ndt_build_and_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -x
+set -e
+
+# Use a clean directory, so that we can mount it from docker run and
+# have access to the finished product, when required.
+mkdir ~/ndt; cd ~/ndt; cp ~/builder/slice-tags.list .
```

Comment:
Yeah - as discussed offline, the up to date slice-tags.list seems to exist only on mlab_builder.  I'll create a PR to update that separately.